org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/ConnectionStreamFactoryTest.java

```diff
@@ -45,6 +47,13 @@ public void testSocketSelection(){
 		System.clearProperty("STDOUT_PORT");
 	}
 
+	@Test
+	public void testStdInOut() throws IOException {
```

Comment:
we want to check System.out.println() doesn't write into tested.getOutputStream(). The test test should fail before applying your change